android/quest/src/test/java/org/smartregister/fhircore/quest/app/fakes/Faker.kt

```diff
@@ -96,4 +96,31 @@ object Faker {
       }
     }
   }
+
+  fun buildTestMeasureReportConfiguration(): ConfigurationRegistry {
+
+    val fhirResourceService = mockk<FhirResourceService>()
+    val fhirResourceDataSource = spyk(FhirResourceDataSource(fhirResourceService))
+    coEvery { fhirResourceService.getResource(any()) } returns Bundle()
+
+    val configurationRegistry =
+      spyk(
+        ConfigurationRegistry(
+          fhirEngine = mockk(),
+          fhirResourceDataSource = fhirResourceDataSource,
+          sharedPreferencesHelper = mockk(),
+          dispatcherProvider = mockk(),
+          configService = mockk(),
+        )
+      )
+
+    runBlocking {
+      configurationRegistry.loadConfigurations(
+        appId = APP_DEBUG,
+        context = ApplicationProvider.getApplicationContext()
+      ) {}
+    }
+
+    return configurationRegistry
+  }
 }
```

Comment:
This is a duplicate of [Faker#L33](https://github.com/opensrp/fhircore/blob/7781e177fc4a60216172d1893fa64c883b1f8b82/android/quest/src/androidTest/java/org/smartregister/fhircore/quest/Faker.kt#L33)